bin/upload-config-to-heroku.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/* Take a taskcluster.base.config config file and generate a command that will
+   upload all values to heroku */
```

Comment:
Do we really need this? Your local config should almost never the same as production.
And it's rarely that many variables to overwrite.. (Anyways, we still own it to rethink config system)